tools/c7n_org/scripts/orgaccounts.py

```diff
@@ -119,7 +121,7 @@ def get_sub_ous(client, ou):
     return results
 
 
-def get_accounts_for_ou(client, ou, active, recursive=True):
+def get_accounts_for_ou(client, ou, active, recursive=True, ignoredAccounts=[]):
```

Comment:
generally its bad practice to ever use a mutable as a default argument for a function, as that variable is retained

ie.

```python
def foo(x=[]):
     x.append(1)
     print(x)

foo()
# prints [1]
foo()
# prints [1, 1]
```

while we're not mutating the arg here, let's just use an empty tuple `ignoreAccounts=()` which is immutable.